### PR TITLE
WiresharkFoundation.Wireshark version 3.6.8

### PR DESCRIPTION
**Pull request by mivsvit**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/78895)

### CODE REVIEW

File: manifests/w/WiresharkFoundation/Wireshark/3.6.8/WiresharkFoundation.Wireshark.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: WiresharkFoundation.Wireshark
+PackageVersion: 3.6.8
```
Comment (Trenly) [inline manifests/w/WiresharkFoundation/Wireshark/3.6.8/WiresharkFoundation.Wireshark.installer.yaml:5]:
```suggestion
PackageVersion: 3.6.8.0
```
This should match Control Panel

File: manifests/w/WiresharkFoundation/Wireshark/3.6.8/WiresharkFoundation.Wireshark.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: WiresharkFoundation.Wireshark
+PackageVersion: 3.6.8
```
Comment (Trenly) [inline manifests/w/WiresharkFoundation/Wireshark/3.6.8/WiresharkFoundation.Wireshark.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.6.8.0
```
This should match Control Panel

File: manifests/w/WiresharkFoundation/Wireshark/3.6.8/WiresharkFoundation.Wireshark.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: WiresharkFoundation.Wireshark
+PackageVersion: 3.6.8
```
Comment (Trenly) [inline manifests/w/WiresharkFoundation/Wireshark/3.6.8/WiresharkFoundation.Wireshark.yaml:5]:
```suggestion
PackageVersion: 3.6.8.0
```
This should match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by mivsvit** [comment]
Will resubmit with requested version number

**Pull request closed without merging**